### ISSUE TITLE
CPX and BND types missing functional annotations

### ISSUE THREAD

**Issue by alsanju** [broadinstitute/gatk-sv#100]
### Affected module(s) or script(s)
Module 08 - Annotation

### Description 
Functional annotations currently missing for some CPX dupINVdup types after running GATK-SV cohort mode. `PROTEIN_CODING_`, `LINCRNA_` and `NONCODING_` fields are not being annotated, even if the variants overlap features present in these annotation sources.

#### Actual behavior
The following SVs are some examples of dupINVdup involving multiple protein coding genes and other features where the annotations, including the `PROTEIN_CODING__INV_SPAN` field, are missing. The post-annotation entries in the VCF are:

```
CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
chr7	66357813	.	N	<CPX>	1	UNRESOLVED	END=68470193;SVTYPE=CPX;CHR2=chr7;SVLEN=2112380;ALGORITHMS=manta;EVIDENCE=PE,SR;CPX_INTERVALS=DUP_chr7:66357813-66603913,INV_chr7:66357813-68470193,DUP_chr7:68180053-68470193;CPX_TYPE=dupINVdup;UNRESOLVED_TYPE=POSTHOC_RD_GT_REJECTION;AN=1428;AC=1;AF=0.0007;N_BI_GENOS=714;N_HOMREF=713;N_HET=1;N_HOMALT=0;FREQ_HOMREF=0.998599;FREQ_HET=0.00140056;FREQ_HOMALT=0;MALE_AN=772;MALE_AC=0;MALE_AF=0;MALE_N_BI_GENOS=386;MALE_N_HOMREF=386;MALE_N_HET=0;MALE_N_HOMALT=0;MALE_FREQ_HOMREF=1;MALE_FREQ_HET=0;MALE_FREQ_HOMALT=0;FEMALE_AN=656;FEMALE_AC=1;FEMALE_AF=0.001524;FEMALE_N_BI_GENOS=328;FEMALE_N_HOMREF=327;FEMALE_N_HET=1;FEMALE_N_HOMALT=0;FEMALE_FREQ_HOMREF=0.996951;FEMALE_FREQ_HET=0.00304878;FEMALE_FREQ_HOMALT=0
chr9	83811757	.	N	<CPX>	692	UNRESOLVED	END=85826526;SVTYPE=CPX;CHR2=chr9;SVLEN=2014769;ALGORITHMS=manta;EVIDENCE=SR;CPX_INTERVALS=DUP_chr9:83811757-83902762,INV_chr9:83811757-85826526,DUP_chr9:85812713-85826526;CPX_TYPE=dupINVdup;UNRESOLVED_TYPE=POSTHOC_RD_GT_REJECTION;AN=1428;AC=21;AF=0.014706;N_BI_GENOS=714;N_HOMREF=694;N_HET=19;N_HOMALT=1;FREQ_HOMREF=0.971989;FREQ_HET=0.0266106;FREQ_HOMALT=0.00140056;MALE_AN=772;MALE_AC=11;MALE_AF=0.014249;MALE_N_BI_GENOS=386;MALE_N_HOMREF=375;MALE_N_HET=11;MALE_N_HOMALT=0;MALE_FREQ_HOMREF=0.971503;MALE_FREQ_HET=0.0284974;MALE_FREQ_HOMALT=0;FEMALE_AN=656;FEMALE_AC=10;FEMALE_AF=0.015244;FEMALE_N_BI_GENOS=328;FEMALE_N_HOMREF=319;FEMALE_N_HET=8;FEMALE_N_HOMALT=1;FEMALE_FREQ_HOMREF=0.972561;FEMALE_FREQ_HET=0.0243902;FEMALE_FREQ_HOMALT=0.00304878
chr10	27042549	.	N	<CPX>	1	UNRESOLVED	END=28567795;SVTYPE=CPX;CHR2=chr10;SVLEN=1525246;ALGORITHMS=manta;EVIDENCE=PE,SR;CPX_INTERVALS=DUP_chr10:27042549-27071432,INV_chr10:27042549-28567795,DUP_chr10:28501475-28567795;CPX_TYPE=dupINVdup;UNRESOLVED_TYPE=POSTHOC_RD_GT_REJECTION;AN=1428;AC=4;AF=0.002801;N_BI_GENOS=714;N_HOMREF=710;N_HET=4;N_HOMALT=0;FREQ_HOMREF=0.994398;FREQ_HET=0.00560224;FREQ_HOMALT=0;MALE_AN=772;MALE_AC=0;MALE_AF=0;MALE_N_BI_GENOS=386;MALE_N_HOMREF=386;MALE_N_HET=0;MALE_N_HOMALT=0;MALE_FREQ_HOMREF=1;MALE_FREQ_HET=0;MALE_FREQ_HOMALT=0;FEMALE_AN=656;FEMALE_AC=4;FEMALE_AF=0.006098;FEMALE_N_BI_GENOS=328;FEMALE_N_HOMREF=324;FEMALE_N_HET=4;FEMALE_N_HOMALT=0;FEMALE_FREQ_HOMREF=0.987805;FEMALE_FREQ_HET=0.0121951;FEMALE_FREQ_HOMALT=0
```

#### Expected behavior
The SVs should be properly annotated and include the corresponding `PROTEIN_CODING_`, `LINCRNA_` and `NONCODING_` information.

**Comment by epiercehoffman** [comment]
To make a record of our offline conversation - all of these events and those in #99 are marked as `UNRESOLVED`, which causes them to be skipped by our current annotation script. This behavior is intentional. But, I will leave these tickets open for now and aim to provide more details and next steps soon.

**Comment by mwalker174** [comment]
@epiercehoffman Will this be addressed with the upcoming tool?

**Comment by epiercehoffman** [comment]
@mwalker174 Yes, the new tool currently annotates CPX and BND variants even if they are marked `UNRESOLVED`. If `UNRESOLVED` variants should be treated differently then we should discuss that further.

**Comment by mwalker174** [comment]
Addressed in #342
